tests/integration/models/device.spec.js

```diff
@@ -2679,6 +2679,9 @@ describe('Device Model', function () {
 			});
 
 			describe('balena.models.device.setSupervisorRelease()', function () {
+				givenADevice(beforeEach, {
+					os_version: 'balenaOS 2.56.1+rev1',
+				});
```

Comment:
NIT: How about just patching the device's os_version instead of creating a new one (on top of the `given a newly registered offline device` adds in its own `beforeEach`?
(Same below)

Reply:
sure, i avoided that as there may be side effects we are bypassing by faking a HUP.

Reply:
I'm fine if you prefer to do it like this (a POST and not a PATCH), I'm only trying to save the duplicate device creation in before each test case.
Feel free to move the whole `describe('balena.models.device.setSupervisorRelease()', function () {` outside the `'given a newly registered offline device'` and just have the `beforeEach` exactly as you prefer it :)

Reply:
done, thanks for the rec!